sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Models/TimeSeriesQueryAnalyzer.cs

```diff
@@ -11,7 +11,7 @@ namespace Azure.IoT.TimeSeriesInsights
     /// <summary>
     /// QueryAnalyzer allows a user to query for pages of data.
```

Comment:
Missed a rename in the comment. I don't think you need to reference the type though. Just say, "Allows a user to query for pages of data."